sdk/iot/Azure.Iot.Hub.Service/src/swagger/iothubservice_modified.json

```diff
@@ -2929,11 +2929,11 @@
           "type": "string"
         },
         "cloudToDeviceMethod": {
-          "$ref": "#/definitions/CloudToDeviceMethod",
+          "$ref": "#/definitions/CloudToDeviceMethodRequest",
```

Comment:
Where is the definition for CloudToDeviceMethodRequest?

Reply:
Looks like I missed changing it. I'll push the fix for that